src/modules/usrloc/ucontact.c

```diff
@@ -1672,6 +1672,7 @@ static inline int update_contact_db(ucontact_t* _c)
  */
 int update_ucontact(struct urecord* _r, ucontact_t* _c, ucontact_info_t* _ci)
 {
+	static struct urecord _ur;
```

Comment:
I don't see a reason to declare the variable _ur static here or in the other cases of the patch. It does not seem to be used outside of the function, so it can be declared locally (on the stack).